src/tox/session/__init__.py

```diff
@@ -220,6 +220,26 @@ def subcommand_test(self):
             retcode = self._summary()
         return retcode
 
+    def _add_parallel_summaries(self):
+        import json
+        if 'testenvs' not in self.resultlog.dict:
+            return
+        for venv in self.venv_dict.values():
```

Comment:
you'll probably need to store the assigned index number on venv_dicts envconfig to be able to look up the correct file

Reply:
please also add a test to validate the feature :

Reply:
Again, the venv.path is unique already.

I will add tests now that I know this approach is fine.

Reply:
No, it's not 🤔 unless you use util - safe file option; e.g. this will fail as the two will step on each other toes (while running them sequentially would work):

```
tox -e py27,py37 -p all --result-json a.out; tox -e py27,py36 -p all b.out 
```

Reply:
Huh, running tox in parallel in the same directory and the same env is actually supported? Wow!

I will look into it then when I'm back later today.

Reply:
I guess with ``from tox.util.lock import get_unique_file``?

Reply:
Yeah, we make sure all log files are uniquely generated via https://github.com/tox-dev/tox/blob/master/src/tox/util/lock.py#L27 (other than this here), so as long as those envs are not modifying the same files (aka are read only) no reason why not to support it 👍

Reply:
I've added ``get_result_json_path`` to VirtualEnv. If this looks good to you, then I'll work on tests tomorrow.